index.d.ts

```diff
@@ -0,0 +1,128 @@
+import { Tuple } from 'ts-toolbelt';
+
+interface PifyOptions {
+    /**
+    By default, the promisified function will only return the second argument from the callback, which works fine for most APIs. This option can be useful for modules like `request` that return multiple arguments. Turning this on will make it return an array of all arguments from the callback, excluding the error argument, instead of just the second argument. This also applies to rejections, where it returns an array of all the callback arguments, including the error.
+
+    @default false
+
+	@example
+	```
+	const request = require('request');
+    const pify = require('pify');
+
+    const pRequest = pify(request, {multiArgs: true});
+
+    (async () => {
+        const [httpResponse, body] = await pRequest('https://sindresorhus.com');
+    })();
+	```
+	*/
+    multiArgs?: boolean,
+
+    /**
+	Methods in a module to promisfy. Remaining methods will be left untouched.
+	*/
+    include?: Array<string | RegExp>,
+
+    /**
+    Methods in a module not to promisify. Methods with names ending with 'Sync' are excluded by default.
+
+	@default [/.+(Sync|Stream)$/]
+	*/
+    exclude?: Array<string | RegExp>,
+
+    /**
+    If the given module is a function itself, it will be promisified. Enable this option if you want to promisify only methods of the module.
+
+    @default false
+
+	@example
+	```
+	const pify = require('pify');
+
+    function fn() {
+        return true;
+    }
+
+    fn.method = (data, callback) => {
+        setImmediate(() => {
+            callback(null, data);
+        });
+    };
+
+    (async () => {
+        // Promisify methods but not `fn()`
+        const promiseFn = pify(fn, {excludeMain: true});
+
+        if (promiseFn()) {
+            console.log(await promiseFn.method('hi'));
+        }
+    })();
+	```
+	*/
+    excludeMain?: boolean,
+
+    /**
```

Comment:
Use tab-indentation

Reply:
Really stupid. I failed at the first and the most obvious style rule.